frontend/www/js/omegaup/graderv2/IDESettings.vue

```diff
@@ -148,80 +153,94 @@ export default class IDESettings extends Vue {
   }
 
   set overallWallTimeLimit(value: number) {
-    store.commit('OverallWallTimeLimit', value.toString());
+    store.dispatch('limits', {
+      ...store.state.request.input.limits,
+      OverallWallTimeLimit: value.toString() + 's',
```

Comment:
nit:
```suggestion
      OverallWallTimeLimit: `${value}s`,
```
It seems that [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#string_interpolation) allows the use of numbers. check this out: